README.md

```diff
@@ -2,16 +2,20 @@
 
 [![CircleCI](https://circleci.com/gh/mana-ethereum/mana.svg?style=svg)](https://circleci.com/gh/mana-ethereum/mana) [![Waffle.io - Columns and their card count](https://badge.waffle.io/mana-ethereum/mana.svg?columns=all)](https://waffle.io/mana-ethereum/mana)[![Gitter chat](https://badges.gitter.im/mana-ethereum/mana.png)](https://gitter.im/mana-ethereum/mana)
 
-Mana-Ethereum is an open-source Ethereum blockchain client built using [Elixir](https://elixir-lang.org/). Elixir runs on the Erlang Virtual Machine, which is used for distributed systems and offers massive scalability and high visibility. These properties make Elixir a perfect candidate for blockchain network development.
+Mana-Ethereum is an open-source Ethereum blockchain client built using [Elixir]. Elixir runs on the Erlang Virtual Machine, which is used for distributed systems and offers massive scalability and high visibility. These properties make Elixir a perfect candidate for blockchain network development.
 
-In the current Ethereum ecosystem, a majority of active nodes on the network are Geth or Parity nodes. Mana-Ethereum provides an additional open-source alternative. Our aim is to create an open, well-documented implementation that closely matches the protocols described in the [Ethereum yellow paper](https://ethereum.github.io/yellowpaper/paper.pdf).
+In the current Ethereum ecosystem, a majority of active nodes on the network are Geth or Parity nodes. Mana-Ethereum provides an additional open-source alternative. Our aim is to create an open, well-documented implementation that closely matches the protocols described in the [Ethereum yellow paper].
 
-Mana-Ethereum is currently in development. See the [Project Status](#project-status) for more information.
+Mana-Ethereum is currently in development. See the [Project Status] for more information.
+
+[Elixir]: https://elixir-lang.org/
+[Ethereum yellow paper]: https://ethereum.github.io/yellowpaper/paper.pdf
+[Project Status]: #project-status
 
 # Dependencies
 
 - Elixir ~> 1.7.3
-- Rust ~> 1.26.0 (as a dependency of [Rox](https://github.com/urbint/rox))
+- Rust ~> 1.26.0 (as a dependency of [Rox])
```

Comment:
we should remove `rust` form dependencies. We don't use `rox` anymore

Reply:
Yup, this line can be removed 👍